File: JavaScript/7kyu/TwoToOne.js
function longest(s1, s2) {
  return Array.from(new Set(s1 + s2))
    .sort()
    .join("");
}

const longest_2 = (s1, s2) => [...new Set(s1 + s2)].sort().join("");

var result = longest_2("loopingisfunbutdangerous", "lessdangerousthancoding");
var expected = "abcdefghilnoprstu";
console.log(result == expected);

//

File: CSharp/7kyu/TwoToOne.cs
using System.Linq;

namespace CSharp._7Kyu
{
    public static partial class Kata7
    {
        public static string Longest(string s1, string s2)
        {
            return string.Join("", (s1 + s2).Distinct().OrderBy(a => a));
        }
        public static string Longest_2(string s1, string s2)
        {
            return string.Concat(s1.Union(s2).Distinct().OrderBy(a => a));
        }
    }
}